BoxContentSDK/BoxContentSDK/Clients/BOXContentClient.m

```diff
@@ -22,6 +22,11 @@
 #import "BOXUserRequest.h"
 #import "BOXContentClient+User.h"
 
+// Default API URLs
+NSString *const BOXDefaultAPIBaseURL = @"https://api.box.com/2.0";
```

Comment:
Makes sense, I will:
1. Add a const for BOXDefaultAPIBaseURLWithoutVersion = https://api.box.com
2. Add class methods 'APIBaseURLWithoutVersion' (getter) and 'setAPIBaseURLWithoutVersion' (setter)